src/Connector.sol

```diff
@@ -132,7 +133,7 @@ contract CentrifugeConnector {
         uint128 price
     ) public onlyRouter {
         Tranche storage tranche = tranches[poolId][trancheId];
-        require(tranche.latestPrice > 0, "CentrifugeConnector/invalid-pool-or-tranche");
+        require(tranche.lastPriceUpdate > 0, "CentrifugeConnector/invalid-pool-or-tranche");
```

Comment:
Although it's very unlikely, token price could someday be 0, and then this check which should just be trying to see if the tranche exists or not, would throw an error. But since we set lastPriceUpdate on tranche creation now, a lastPriceUpdate of 0 will *always* mean the tranche doesn't exist.

Reply:
Very good point 💯